src/usr/local/www/guiconfig.inc

```diff
@@ -1087,28 +1087,34 @@ function alias_info_popup($alias_id) {
 	return $content;
 }
 
-function gateway_info_popup($gateway) {
-	global $config;
-	init_config_arr(array('gateways', 'gateway_item'));
+function gateway_info_popup($showgw) {
+	global $config, $user_settings;
+
+	if ($user_settings['webgui']['disablealiaspopupdetail']) {
```

Comment:
Why key off this? The alias popup detail option is there to suppress them because if a user has gigantic aliases, the popup windows do not work properly and can affect page rendering and speed.  There wouldn't be any similar issues with gateway popups.

Reply:
Ok, removed